packages/@react-spectrum/s2/src/ActionButton.tsx

```diff
@@ -226,6 +227,13 @@ export const btnStyles = style<ButtonRenderProps & ActionButtonStyleProps & Togg
       [iconOnly]: 'calc(self(minWidth)/2 + var(--iconWidth)/2)',
       [textOnly]: 'full'
     }
+  },
+  '--avatarBorderRadius': {
+    type: 'borderTopStartRadius',
+    value: {
+      default: 'full',
+      [avatarOnly]: 'none'
```

Comment:
I thought the issue was that the button no longer had an aspect ratio of 1/1 (square), not that the corners should be square?

Reply:
Ahh that makes more sense. No wonder I couldn't find any designs